src/main/java/core/basesyntax/operationshandlers/OperationHandler.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.operationshandlers;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationHandler {
+    int MIN_AMOUNT = 0;
```

Comment:
I think its not the way magic number should look like, as I remember it should be initialized by "private static final"